### PR TITLE
Enable editing of unsupported blocks

### PR DESCRIPTION
**Pull request by guarani**
Related https://github.com/wordpress-mobile/gutenberg-mobile/pull/2457

To test: See https://github.com/WordPress/gutenberg/pull/23592

PR submission checklist:

- [ ] I have considered adding unit tests where possible.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by novi-kolacic** [comment]
Hey @etoledom 👋 
Should we update release notes?

**Comment by marecar3** [comment]
Hey @etoledom should we update release notes?

**Comment by etoledom** [comment]
Updated this PR deactivating the Unsupported Block Editor feature if the site is connected via Jetpack, using the changes from here: https://github.com/WordPress/gutenberg/pull/23592#issuecomment-652476700

Jetpack-connected sites will be recognised as `wpcom` site by our auth algorithm, but since we use the self-hosted url to load gutenberg, the cookies obtained won't work.

We should request the cookies using the self-hosted way, but being logged-in on Wpcom account, we don't have the self-hosted username and password required for this to work.

While we can't solve this problem, we won't be able to edit unsupported blocks on Jetpack sites.

**Comment by etoledom** [comment]
Steps to test:
- Create a **wpcom** account+site
- Create a **self-hosted** site
- Connect your **self-hosted** site to the **wpcom** account
- Create a second **self-hosted** site.
- Add this last **self-hosted** site to the app as a **self-hosted (non-jetpack)** site.
- Add a new post with an Unsupported block to all three sites.
- check that:
  - **WPCom** site shows the feature and Gutenberg-Web loads as expected.
  - **Self-Hosted** site shows the feature and Gutenberg-Web loads as expected.
  - **Jetpack** site does not show the feature.

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift [hunk 1]
```diff
@@ -869,9 +869,14 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
 
     func gutenbergCapabilities() -> [String: Bool]? {
         return [
-            "mentions": post.blog.isAccessibleThroughWPCom()
+            "mentions": post.blog.isAccessibleThroughWPCom(),
+            "unsupportedBlockEditor": !disableUnsupportedBlockEditor,
         ]
     }
+
+    private var disableUnsupportedBlockEditor: Bool {
```
[the remark this answers is not in the document]
Comment (etoledom) [reply WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift]: Completely agree! Thanks 🙏

**Pull request merged**